### PR TITLE
[MODUSERS-373]-Added indexes and where clause

### PR DESCRIPTION
**Pull request by SinghAdes**
https://issues.folio.org/browse/MODUSERS-373

### CODE REVIEW

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -220,6 +230,10 @@
         {
           "fieldName": "customFields",
           "tOps":"ADD"
+        },
+        {
+          "fieldName" : "type",
+          "tOps" : "ADD"
```
Comment (SerhiiNosko) [inline src/main/resources/templates/db_scripts/schema.json:236]: We don't need fullText and gin index here. 
Just common index, also please make sume testing, create cql query to filter out type='shadow' and make sure that index used.
Firstly you need to check that there is no warning that index missed in mod-users logs.
Secondly you should get generated sql query and invoke EXPLAIN ANALYZE to confirm that index used

Comment (SinghAdes) [reply src/main/resources/templates/db_scripts/schema.json:236]: image

Comment (SinghAdes) [reply src/main/resources/templates/db_scripts/schema.json:236]: image

**Review by SerhiiNosko — APPROVED** [review]

**Review by tatsiana-tarhonskaya — APPROVED** [review]

**Review by Abdulkhakimov — APPROVED** [review]

**Review by gurleenkaurbp — COMMENTED** [review]
Looks good

**Pull request merged**